Owncloud iOs Client/Login/Login/UniversalLoginViewController.swift

```diff
@@ -606,14 +606,12 @@ public enum TextfieldType: String {
 
         case .BASIC_HTTP_AUTH:
             self.resetPasswordFooterMessage()
-            let userCredDto: OCCredentialsDto = OCCredentialsDto()
-            userCredDto.userName = self.textFieldUsername.text
-            userCredDto.accessToken = self.textFieldPassword.text
-            userCredDto.authenticationMethod = authMethod
+            self.userNewCredentials = OCCredentialsDto()
```

Comment:
if this _OCCredentialsDto_ is extrictly needed to be a global variable this is Ok, if not this is not OK.